GiEnJul/Controllers/AdminController.cs

```diff
@@ -89,5 +91,26 @@ public async Task<ActionResult> PostConnectionAsyc(string giverRowKey, string re
             }
             return Ok();
         }
+        [HttpDelete("giver")]
+        public async Task<ActionResult> DeleteGiverAsync([FromBody] DeleteGiverDto giverDto)
+
+        {
+            System.Console.WriteLine("im here");
+            try
+            {
+                System.Console.WriteLine("Trying");
+                var giverToDelete = await _giverRepository.GetGiverAsync(giverDto.PartitionKey, giverDto.RowKey);
+                if (giverToDelete == null)
+                    return NotFound($"Giver with PK = {giverDto.PartitionKey} and RK = {giverDto.RowKey} not found");
+                var deletedGiver = await _giverRepository.DeleteAsync(giverToDelete);
+                return Ok();
+            }
+            catch (System.Exception err)
+            {
+                System.Console.WriteLine(err);
+                return StatusCode(StatusCodes.Status500InternalServerError,
+                "Error deleting data");
+            }
```

Comment:
The behavior of this block is equal to the standard error handler. 
The default handler would write the error to terminal, and return a HTTP status 500. Because of this, i belive its not necessary with a try catch here.

Reply:
So I could remove everything regarding error handling then, and just have line 99, 102 and 103 @Sommerli ?

Reply:
Fixed with 6971c59 @Sommerli